### PR TITLE
Cherry-pick bugfixes for 6.3.2

### PR DESCRIPTION
**Pull request by tellthemachines**
## What?

Cherry-picks the following PRs:

[Fallback to default max viewport if layout wide size is fluid.](https://github.com/WordPress/gutenberg/pull/53551)
[Site editor: add missing i18n in HomeTemplateDetails](https://github.com/WordPress/gutenberg/pull/53543) 
[RichText: Remove 'Footnotes' when interactive formatting is disabled](https://github.com/WordPress/gutenberg/pull/53474) 
[Preserve block style variations when securing theme json](https://github.com/WordPress/gutenberg/pull/53466) 
[Image: Clear aspect ratio when wide aligned](https://github.com/WordPress/gutenberg/pull/53439)
[Fix missing Replace button in content-locked Image blocks](https://github.com/WordPress/gutenberg/pull/53410) 
[Remove "go to" for terms and posts](https://github.com/WordPress/gutenberg/pull/53408)
[Image block: Fix stretched images constrained by max-width](https://github.com/WordPress/gutenberg/pull/53274) 
[Fix: Sync status overlaps for some languages in Patterns post type page](https://github.com/WordPress/gutenberg/pull/53243)
[Fix document title alignment in command palette button](https://github.com/WordPress/gutenberg/pull/53224)
[Update document title buttons radius](https://github.com/WordPress/gutenberg/pull/53221)
[Fix: Snack bar not fixed on certain pages in the Site Editor](https://github.com/WordPress/gutenberg/pull/53207) 
[Image Block: Don't render DimensionsTool if it is not resizable ](https://github.com/WordPress/gutenberg/pull/53181)
[Site Editor: Fix document actions label helper method](https://github.com/WordPress/gutenberg/pull/52974)
[Add tests for fluid layout + typograph](https://github.com/WordPress/gutenberg/pull/53554)
[Fix support of sticky position in non-iframed post editor](https://github.com/WordPress/gutenberg/pull/53540) 
[Link Control: persist advanced settings toggle state to preferences if available](https://github.com/WordPress/gutenberg/pull/52799)
[Fix: indicator style when block moving mode](https://github.com/WordPress/gutenberg/pull/53972)
[Fix post editor top toolbar with custom fields in Safari](https://github.com/WordPress/gutenberg/pull/53688)
[Set top toolbar size dynamically](https://github.com/WordPress/gutenberg/pull/53526)

Edit: getting these fixes to core will be delayed a bit due to timing with WCUS around the corner. In the meantime, 6.3.1 is planned with only a few high-priority fixes. To that end, I reverted the inclusion of [Footnotes: Fix recursion into updating attributes when attributes is not an object](https://github.com/WordPress/gutenberg/pull/53257) in this PR and have added it instead to #53709 so it can go into 6.3.1.

### CODE REVIEW

**Comment by ramonjd** [comment]
I think this PR is missing:

- https://github.com/WordPress/gutenberg/pull/53554

It might fix the JS unit tests

**Review by ramonjd — APPROVED** [review]
Tested all commits LGTM.

Just needs the tests from https://github.com/WordPress/gutenberg/pull/53554 I think

**Comment by ramonjd** [comment]
🤞🏻 we could get the following tested and merged as well?

- https://github.com/WordPress/gutenberg/pull/53660

**Comment by tellthemachines** [comment]
>  we could get the following tested and merged as well?

Sure, I wasn't planning to publish packages straightaway, was going to wait until #53596 is merged too

**Comment by gziolo** [comment]
Noting that it’s very likely that npm publishing will have to be executed manually from a local machine directly in the `wp/6.3` branch because of:
- https://github.com/WordPress/gutenberg/issues/53628

**Comment by tellthemachines** [comment]
> Noting that it’s very likely that npm publishing will have to be executed manually from a local machine directly in the `wp/6.3`

oh joy 😭

**Comment by gziolo** [comment]
> Noting that it’s very likely that npm publishing will have to be executed manually from a local machine directly in the `wp/6.3` branch because of:
> 
> * [Packages: Publishing to npm fails for `wp/X.Y` branches because of older Node.js #53628](https://github.com/WordPress/gutenberg/issues/53628)

As I anticipated, the job for npm publishing failed because of npm version difference between branches:

https://github.com/WordPress/gutenberg/actions/runs/5874702589/job/15929869480#step:8:409

Do you need help with publishing to npm?

**Comment by gziolo** [comment]
https://github.com/WordPress/gutenberg/pull/53762 should help with npm publishing targeting WP 6.3.

**Pull request merged**
